src/interfaces/ICoWSwapEthFlow.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8;
+
+import "../vendored/GPv2Order.sol";
+
+/// @title CoW Swap ETH Flow Interface
+/// @author CoW Swap Developers
+interface ICoWSwapEthFlow {
+    /// @dev Error thrown when trying to create a new order whose order hash is the same as an order hash that was
+    /// already assigned.
+    error OrderIsAlreadyOwned(bytes32 orderHash);
+
+    /// @dev Error thrown when trying to create an order without sending the expected amount of ETH to this contract.
+    error IncorrectEthAmount();
+}
```

Comment:
Should `function createOrder(EthFlowOrder.Data calldata order, uint64 quoteId)` and `EthFlowOrder.Data` also be part of the interface?

Reply:
Absolutely! I didn't do it at first because I thought I needed to have this interface inherited by the test contract to have access to the custom errors (like it's the case for events) but this didn't turn out to be necessary.